src/scenes/App/components/TermsAndConditions/index.jsx

```diff
@@ -0,0 +1,274 @@
+              {this.renderRepresentations()}
+              {this.renderAwards()}
+              {this.renderPowers()}
+              {this.renderResponsibilities()}
+              {this.renderIp()}
+              {this.renderSupport()}
+              {this.renderMisc()}
+            </Paper>
+            <FormGroup row className={classes.formGroup}>
+              <FormControlLabel
+                control={
+                  <Checkbox
+                    checked={checkboxChecked}
+                    value="checkboxChecked"
+                    onChange={this.handleCheckedChange('checkboxChecked')}
+                  />
+                }
+                label={<FormattedMessage id="tnc.acceptCheckboxText" defaultMessage="I accept the Terms and Conditions" />}
+              />
+              <Button
+                type="submit"
+                color="primary"
+                variant="raised"
+                disabled={!checkboxChecked}
+                className={classes.acceptButton}
+                onClick={this.onAcceptButtonClick}
+              >
+                <FormattedMessage id="tnc.accept" defaultMessage="Accept" />
+              </Button>
+            </FormGroup>
+          </Grid>
+          <Grid item xs></Grid>
+        </Grid>
+      </div>
+    );
+  }
+
+  handleCheckedChange = (name) => (event) => {
+    this.setState({ [name]: event.target.checked });
+  }
+
+  onAcceptButtonClick = () => {
+    localStorage.setItem(TNC_ACCEPTED, true);
+    this.forceUpdate();
+  }
+
+  renderTitle = () => (
+    <Typography variant="headline" className={this.props.classes.title}>
+      <FormattedMessage id="tnc.title" defaultMessage="Bodhi Terms and Conditions" />
+    </Typography>
+  );
+
+  renderGeneral = () => (
+    <div>
+      <Typography variant="title">
+        <FormattedMessage id="tnc.general" defaultMessage="General" />
+      </Typography>
+      <ol>
+        <li><FormattedMessage id="tnc.general1" defaultMessage="The Site is operated by the Bodhi Foundation (“Bodhi“), with registered address at 22 NORTH CANAL ROAD #02-00 SINGAPORE (048834). At this stage of the Platform development this Site is entirely for free, and no risking of any BOT or QTUM Tokens is required in order to use it. Therefore, all Events and/or Predictions made via the Site are entirely risk free, however this is subject to change in accordance with the development milestones in the White Paper." /></li>
+        <li><FormattedMessage id="tnc.general2" defaultMessage="These terms and conditions (“Terms and Conditions“) govern your (“You“, “Your”, “Player“ or “Participant”) use of the platform provided to You by Bodhi.  These Terms and Conditions should be read carefully by You in their entirety prior to Your use of the Platform and/or Site. Please note that these Terms and Conditions constitute a legally binding agreement between You and the Company." /></li>
+      </ol>
+    </div>
+  );
+
+  renderDefinitions = () => (
```

Comment:
This should be a component, not a method. Make a stateless component please.

Reply:
done